test/integration/operator_olm_test.go

```diff
@@ -37,7 +37,7 @@ import (
 )
 
 const (
-	defaultTimeout = 2 * time.Minute
+	defaultTimeout = 4 * time.Minute
```

Comment:
Any specfic reason for increasing the timeout?

Reply:
The 2 minute timeout caused two consecutive failures in CI.

Other e2e tests use 4 minutes for `run packagemanifests`, so I chose that.

Reply:
Makes sense 👍